### PR TITLE
Bugfix/parent dashboard form standardization

### PR DESCRIPTION
**Pull request by Alisa1989**
In this Pull Request the styling for 4 different components was made more consistent.
This was necessary because the pages didn't look like they belonged to the same website.

Description:
- I initially made sure that the header on each page was consistent, one of them had different styling
- one of the pages was missing the "go back to dashboard component" which I added
- made changes to the general structures of the Forms so they could more easily look consistent
- once that was done I used the same class name for certain elements so the aesthetics looked the same
- modified colors throughout the styling files to colors that are approved in the project [schema](https://www.figma.com/file/U9Lo9qjZgl4Vbgb2cCMq2v/Story-Squad---Labs?node-id=522%3A1456)

- [x] Make sure input labels are uniform(same font, location, text size)
- [ ] Make sure text box are uniform(same size, shape)
- [x] Keep buttons consistent(same location, color, size and shape)

Want didn't happen:
- I wasn't able to change the size of the text boxes for the "parent/settings" page

[Trello Card](https://trello.com/c/gha0vi2A)

https://user-images.githubusercontent.com/37967343/139164389-2bcc5655-7593-4c0d-90fb-9bfcb7f7deab.mp4

### CODE REVIEW

File: src/components/pages/EditPlayers/RenderEditPlayers.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const EditPlayers = props => {
       <Layout className="content">
         <div className="top-section">
           <ParentDashboardBack />
-          <Title className="title" level={2}>
+          <Title className="title" style={{ color: '#2D2A2B' }} level={2}>
```
Comment (just-kato) [inline src/components/pages/EditPlayers/RenderEditPlayers.js]: Nice work with this color change!

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const FamilySettings = props => {
     <Layout className="parent-dashboard add-child">
       <ParentNavTopBar selected="settings" />
       <Layout className="content">
-        <Title className="title" style={{ color: '#0267C1' }} level={1}>
+        <Title className="title" style={{ color: '#2D2A2B' }} level={1}>
```
Comment (just-kato) [inline src/components/pages/FamilySettings/RenderFamilySettings.js]: Looks great. I remember we talked about this refactor.

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ const ParentContactContainer = () => {
           <Row className="card">
             <Col span={16}>
               <Row className="title">
-                <h4 style={{ color: '#878D93', fontWeight: '700' }}>
+                <h4 style={{ color: '#2D2A2B', fontWeight: '700' }}>
```
Comment (just-kato) [inline src/components/pages/ParentContact/ParentContactContainer.js]: Great work!

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ const ParentContactContainer = () => {
   });
   return (
     <>
-      <Layout className="newparent-dashboard">
+      <Layout>
```
Comment (just-kato) [inline src/components/pages/ParentContact/ParentContactContainer.js]: Nice job with keeping this component consistent with the others

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 5]
```diff
@@ -29,12 +29,14 @@ const FamilySettings = props => {
...
 
   return (
-    <Layout>
+    <Layout className="parent-dashboard">
       <ParentNavTopBar selected="settings" />
       <Layout className="content">
-        <Title className="title" style={{ color: '#2D2A2B' }} level={1}>
-          Settings
-        </Title>
+        <div className="top-section">
```
Comment (just-kato) [inline src/components/pages/FamilySettings/RenderFamilySettings.js:36]: Nice work

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 6]
```diff
@@ -29,7 +29,7 @@ const ParentContactContainer = () => {
   });
   return (
     <>
-      <Layout>
+      <Layout className="parent-dashboard">
```
Comment (just-kato) [inline src/components/pages/ParentContact/ParentContactContainer.js]: Nice job

File: src/styles/less/ParentDashboard.less [hunk 7]
```diff
@@ -6,69 +6,81 @@
...
-      .ant-layout {
-        background: none;
-      }
-      img {
-        size: 40%;
-        height: 22vh;
-        padding: 10%;
+    .top-section, .settings-form {
+      width: 75%;
+      max-width: 800px;
```
Comment (just-kato) [inline src/styles/less/ParentDashboard.less]: Huge blocks of code deleted and simplified. Nice work

File: src/components/pages/AddChild/RenderAddChild.js [hunk 8]
```diff
@@ -11,7 +11,7 @@ const { Title } = Typography;
 
 const RenderAddChild = props => {
   return (
-    <Layout className="add-child">
+    <Layout className="edit-players">
```
Comment (just-kato) [inline src/components/pages/AddChild/RenderAddChild.js:14]: Nice work with the classname change for the refactor

File: src/components/pages/EditPlayers/RenderEditPlayers.js [hunk 9]
```diff
@@ -15,7 +15,7 @@ const EditPlayers = props => {
       <Layout className="content">
         <div className="top-section">
           <ParentDashboardBack />
-          <Title className="title" style={{ color: '#2D2A2B' }} level={2}>
```
Comment (just-kato) [inline src/components/pages/EditPlayers/RenderEditPlayers.js]: Great job

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 10]
```diff
@@ -29,15 +29,13 @@ const ParentContactContainer = () => {
   });
   return (
     <>
-      <Layout className="parent-dashboard">
+      <Layout className="edit-players">
```
Comment (just-kato) [inline src/components/pages/ParentContact/ParentContactContainer.js]: Nice work!

File: src/styles/less/EditPlayers.less [hunk 11]
```diff
@@ -66,7 +66,7 @@
                         min-width: 115px;
                         padding: 1%;
                         clip-path: circle();
-                        background-color: white;
+                        background-color: #F5F5F5;
```
Comment (just-kato) [inline src/styles/less/EditPlayers.less:72]: Love to see the consistency

File: src/styles/less/ParentDashboard.less [hunk 12]
```diff
@@ -6,81 +6,69 @@
...
-      margin-bottom: 37px;
-      .title {
-          .titleStyle();
-          display: block;
-      }
+  .heading-section {
+    width: 69vw;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
```
Comment (just-kato) [inline src/styles/less/ParentDashboard.less]: Nice work

File: src/styles/less/ParentDashboard.less [hunk 13]
```diff
@@ -6,81 +6,69 @@
...
+      height: 30vh;
+      margin: 0 2.5vw 2.5vh 0;
+      padding: 16px;
+      width: 45%;
+      border-radius: 14px;
+      justify-content: center;
+      text-align: center;
 
-  //     h2 {
-  //       align-self: center;
```
Comment (just-kato) [inline src/styles/less/ParentDashboard.less]: Deleting this commented out code is huge! Easier on the eyes. Nice work

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 14]
```diff
@@ -32,98 +32,127 @@ const FamilySettings = props => {
...
-              name={child.Name}
-              AvatarURL={child.AvatarURL}
-              update="SETTINGS"
-            />
-          ))}
-        </Layout>
 
-        <Layout className="settings-form">
-          <h2 className="h2-family-settings">Change Email or Password</h2>
+      <div className="top-section">
```
Comment (just-kato) [inline src/components/pages/FamilySettings/RenderFamilySettings.js]: great refactor

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 15]
```diff
@@ -32,98 +32,127 @@ const FamilySettings = props => {
...
-                ]}
-              >
-                <Input placeholder="Re-enter New Email" />
-              </Form.Item>
-            </Form.Item>
-            <Form.Item wrapperCol={{ span: 20, offset: 8 }}>
-              <Button
-                style={{ backgroundColor: '#007AFF', color: 'white' }}
-                type="primary"
+            <Row className="form-inputs">
```
Comment (just-kato) [inline src/components/pages/FamilySettings/RenderFamilySettings.js]: Nice job

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 16]
```diff
@@ -28,92 +28,87 @@ const ParentContactContainer = () => {
...
-      <Layout className="edit-players">
-        <ParentNavTopBar />
-        <div className='"site-card-border-less-wrapper"'>
-          <Row className="card">
-            <Col span={16}>
-              <Row className="title">
-                <h4>CONTACT US</h4>
+    <Layout className="edit-players">
+      <ParentNavTopBar />
+      <div className='"site-card-border-less-wrapper"'>
```
Comment (just-kato) [inline src/components/pages/ParentContact/ParentContactContainer.js:33]: Great work!

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 17]
```diff
@@ -32,127 +32,115 @@ const FamilySettings = props => {
...
-      </div>
-      <Layout className="children" style={{ flexFlow: 'row wrap' }}>
-        {props.parent.children.map(child => (
-          <ChildCard
-            name={child.Name}
-            AvatarURL={child.AvatarURL}
-            update="SETTINGS"
-          />
-        ))}
+        <Layout className="children">
```
Comment (just-kato) [inline src/components/pages/FamilySettings/RenderFamilySettings.js:52]: Nice work with this form

File: src/styles/less/NewChildCard.less [hunk 18]
```diff
@@ -115,7 +115,7 @@
       margin-top:-3%;
       font-size: 1.5rem;
       text-align:center;
-      -webkit-text-stroke: 0.1px white;
+      -webkit-text-stroke: 0.1px #FBFBFB;
```
Comment (just-kato) [inline src/styles/less/NewChildCard.less:118]: Nice job with keeping the colors consistent

File: src/styles/less/EditPlayers.less [hunk 19]
```diff
@@ -121,11 +125,15 @@
                             }
                         }
                         input {
-                            .inputStyle();
+                            // .inputStyle();
+                            border: 0.5px solid #7b8794;
```
Comment (just-kato) [inline src/styles/less/EditPlayers.less:129]: great work

File: src/styles/less/EditPlayers.less [hunk 20]
```diff
@@ -75,7 +78,7 @@
                         display: inline-block;
 
                         background-color: #F5F5F5;
-                        border: 1px solid #E5E5E5;
+                        border: 1px solid #F5F5F5;
```
Comment (just-kato) [inline src/styles/less/EditPlayers.less:81]: Nice!

File: src/styles/less/EditPlayers.less [hunk 21]
```diff
@@ -35,12 +35,16 @@
...
         align-items: center;
         .child-card-form {
-            background-color: #F5F5F5;
-            width: 100%;
-            height: fit-content;
-            margin-right: 0;
-            margin-bottom: 30px;
-            padding: 3% 5%;
+              width: 66%;
+                background-color: #fbfbfb;
```
Comment (just-kato) [inline src/styles/less/EditPlayers.less:39]: Great job

**Review by just-kato — DISMISSED** [review]

**Review by just-kato — APPROVED** [review]

File: src/components/pages/ParentContact/ParentContactContainer.js [hunk 22]
```diff
@@ -28,94 +28,87 @@ const ParentContactContainer = () => {
...
+        <Row className="card">
+          <Col span={16}>
+            <Row className="title">
+              <h4>CONTACT US</h4>
+            </Row>
+            <div className="description">
+              <Row>
+                <p>
+                  If you would like to let us know how we're doing of if you are
```
Comment (tonysorensen) [inline src/components/pages/ParentContact/ParentContactContainer.js:42]: Fix the type here. "of" should be "or" also "bellow" should be "below"

File: src/styles/less/EditPlayers.less [hunk 23]
```diff
@@ -110,22 +113,27 @@
...
                                 display: none;
                             }
                         }
                         input {
-                            .inputStyle();
+                            // .inputStyle();
+                            border: 0.5px solid #7b8794;
+                            border-radius: 16px;
+                            padding: 7px;
+                            font-size: 16px;
```
Comment (tonysorensen) [inline src/styles/less/EditPlayers.less:132]: I'm a big fan of responsive units, especially for font-size. Just a consideration, I'm not familiar with how ant design handles their components.

File: src/components/pages/FamilySettings/RenderFamilySettings.js [hunk 24]
```diff
@@ -29,12 +30,15 @@ const FamilySettings = props => {
   };
 
   return (
-    <Layout className="parent-dashboard add-child">
+    <Layout className="edit-players">
```
Comment (tonysorensen) [inline src/components/pages/FamilySettings/RenderFamilySettings.js:33]: Looking at what the component does this seems to make more sense than the previous class name.

File: src/components/pages/AddChild/RenderAddChild.js (hunk 8 above)
Comment (tonysorensen) [reply src/components/pages/AddChild/RenderAddChild.js:14]: I take it the plan is to just consider all additions and edits to be "edit"s and that is the reason for the class name change? I realize it is just a CSS class name but it does catch the eye in an Add component.

**Review by tonysorensen — APPROVED** [review]

File: src/components/pages/EditPlayers/RenderEditPlayers.js (hunk 1 above)
Comment (bipin-shrestha) [reply src/components/pages/EditPlayers/RenderEditPlayers.js]: Good work updating the color.

File: src/styles/less/EditPlayers.less (hunk 23 above)
Comment (bipin-shrestha) [reply src/styles/less/EditPlayers.less:132]: It is going to be interesting to see the change  :)

**Comment by bipin-shrestha** [comment]
Thanks for taking time to make the changes Alisa1989.

**Review by bipin-shrestha — APPROVED** [review]
Thanks for taking time to make changes Alisa1989.

**Review by kendra4227 — COMMENTED** [review]
Nice clean up on the forms

**Pull request merged**